openbb_platform/core/openbb_core/app/static/package_builder.py

```diff
@@ -500,7 +501,15 @@ def get_extra(field: FieldInfo) -> dict:
         """Get json schema extra."""
         field_default = getattr(field, "default", None)
         if field_default:
-            return getattr(field_default, "json_schema_extra", {})
+            keys_to_remove = ["choices"]
+
+            json_schema_extra = {
+                k: v
+                for k, v in getattr(field_default, "json_schema_extra", {}).items()
+                if k not in keys_to_remove
+            }
+
+            return json_schema_extra
```

Comment:
Here we can avoid the iterations

```python
json_schema_extra = getattr(field_default, "json_schema_extra", {})
json_schema_extra.pop("choices", None)
return json_schema_extra
```

Reply:
Was thinking that more keys could be added to `keys_to_remove` in the future, but yes we can do that and eventually to smt like this if needed!